MAINTAINERS.md

```diff
@@ -0,0 +1,122 @@
+**don't create a new release manually**.
+
+It's recommended that discourse on development related topics is held directly
+within GitHub as much as possible. This allows a public commentary on changes,
+providing transparency for all users. The communication channel should mainly be
+used for development related questions. If it becomes clear that a disagreement
+of opinion exists, make sure a public note is placed on the respective issue or
+PR on Github.
+
+All maintainers are responsible for carrying out reviews of both issues and PR's.
+When providing feedback be civil, courteous, and kind. Disagreement is fine, so
+long as the discourse is carried out politely. It is highly recommended that
+reviews are performed by maintainers that have not been involved with the issue
+earlier.
+
+During your review, consider the following points:
+
+* Does the change have impact? While fixing typos is nice as it adds to the
+  overall quality of the project, merging a single typo fix at a time clutters
+  the git history and makes tracing back the origin of a change more difficult.
+  Other examples to be wary of:
+    * Changes in variable names. Ask whether or not the change will make
+      understanding the code easier, or if it could simply a personal preference
+      on the part of the author.
+    * Formatting changes. Formatting changes should usually be generated only by
+      the coding standards checker/fixer. Ask whether the change is generally
+      improving readability/maintenance, or if it could simply be a personal
+      preference on the part of the author.
+  Essentially: feel free to close issues or PR's that do not have impact.
+* Do the changes make sense? If you do not understand what the changes are or
+  what they accomplish, ask the author for clarification. Ask the author to add
+  comments and/or clarify test case names to make the intentions clear. If these
+  clarification reveal that the author may be unaware of features that already
+  accommodate their needs, show an example on how to address the issue with the
+  current features and feel free to close the issue once they confirm.
+* Does the change break backwards compatibility (BC)? In general, no data should
+  be lost. If so, work with the contributor to determine why the BC break is
+  needed, and whether or not there may be a way to make the change without
+  breaking BC. Any BC breaks you plan on merging MUST be discussed with the
+  maintainer team prior to merging.
+* Is this a new feature? If so:
+    * If no issue is linked, does the PR contain information indicating the need
+      for the feature? If not, ask them to provide that information, preferably
+      in an issue.
+    * Are the new tests actually meaningful? Do they sensibly test the behaviour,
+      similarly to how a user would experience it? If not, **do not merge** the
+      feature until they are!
+    * Is the feature necessary for general use cases? Kiwi isn't, nor should it
```

Comment:
like we talked about, this sentence could do with a rewrite to make it easier to read